src/components/MembershipForm/Buttons/SubmitButton.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styled from "styled-components";
+
+const Button = styled.button`
+  color: ${(props) => props.theme.secondary};
+  background: ${(props) => props.theme.primary};
+  font-size: 12px;
+  font-weight: bold;
+  padding: 12px 55px;
+  margin: 20px;
+  border-radius: 20px;
```

Comment:
the border radius of the parent element and the child should be same, make them something 
around 10px